src/fpu/fpu_instructions_x86.h

```diff
@@ -503,7 +502,9 @@
 #else
 
 // !defined _MSC_VER
-
+#if defined(_MSC_VER) && !defined(__clang__)
+#error "Requires Clang under Visual Studio"
+#endif
```

Comment:
Does this mean we're dropping support for the MSVC compiler?  Seems a bit heavy handed.  Can we not just disable the x87 features (and maybe log a warning)?

Reply:
I'm not sure why we'd continue to support MSVC when it doesn't meet our needs? Clang is fully supported and provides a superset of MSVC's features.

Reply:
Some people might not want to install another 8-9GB of dependencies...

Also, I noticed when compiling, the output window doesn't seem to be as feature rich as when compiling with `cl`, making it harder to diagnose build failures.

Reply:
Not too concerned with disk space usage, I'm more concerned about someone developing/debugging with one toolchain, then the release build going out with a different toolchain. The main point of this PR is to consolidate to one supported toolchain instead of "well, someone might like to use other toolchain".

Reply:
Disk space requirements for installing the required toolchain are moot points. If you want to develop Staging on platform X, you need to install the required toolchain, period.

MSYS2 on Windows and XCode on macOS would also install gigabytes of stuff. That's just the way it is, that's the entry ticket for dev work.

I'm fully in agreement with @kklobe -- we should support _one toolchain_ per platform that produces the one build we ship. 

We're a small team and I think we've already spread ourselves too thin with too many build types / toolchains per platform. As discussed, cutting this back to the minimum per platform (what we _actually need_ to deliver our software) is an important step so we can focus more on the actual dev work and less on babysitting the plumbing.

Having two different Windows builds with _different features_ (!) has always bothered the hell out of me. That was confusing for users, so we got rid of the gimped MSVC build. But now that @kklobe has made MSVC full featured, I see zero value in keeping the gimped variant around. That's just asking for trouble and we want to simplify, not complicate 😄